hack/update-istio.sh

```diff
@@ -2,14 +2,31 @@
 
 set -euo pipefail
 
+SLEEP_TIME=60
```

Comment:
Can we reduce this to 10s or less, so that folks actually see that something's happening? 1minute seems excessive and will look like the script has hanged.